src/Compilers/Core/MSBuildTask/Microsoft.VisualBasic.Core.targets

```diff
@@ -51,7 +51,6 @@
          ChecksumAlgorithm="$(ChecksumAlgorithm)"
          CodeAnalysisRuleSet="$(ResolvedCodeAnalysisRuleSet)"
          CodePage="$(CodePage)"
-         CompilerType="$(RoslynCompilerType)"
```

Comment:
Consider leaving a comment on `ManagedCompiler.CompilerType` and removing the logic it contains